js/style/style_layer/fill_extrusion_style_layer.js

```diff
@@ -4,6 +4,15 @@ const StyleLayer = require('../style_layer');
 const FillExtrusionBucket = require('../../data/bucket/fill_extrusion_bucket');
 
 class FillExtrusionStyleLayer extends StyleLayer {
+
+    getPaintValue(name, globalProperties, featureProperties) {
+        const value = super.getPaintValue(name, globalProperties, featureProperties);
+        if (name === 'fill-extrusion-color') {
+            value[3] = 1;
```

Comment:
The reason for this is that if you provide an rgba color in the style, we premultiply it in StyleDeclaration, but we don't render using the alpha channel, so providing a non-opaque rgba value leads to unexpected color results. We decided the best of several not-great approaches here would be to deliberately ignore the alpha channel before premultiplying.